packages/tools/devtools/devtools-example/src/FluidObject.ts

```diff
@@ -222,11 +223,11 @@ export class AppData extends DataObject {
 		const stringSchema = builder.leaf("string-property", ValueSchema.String);
 		const numberSchema = builder.leaf("number-property", ValueSchema.Number);
 		const booleanSchema = builder.leaf("boolean-property", ValueSchema.Boolean);
```

Comment:
Again I don't know why this example has a todo to use a handle, but I'm keeping it.

Reply:
@CraigMacomber I think @jikim-msft left this here based on your recommendation. Adding a handle to another DDS was not previously working, but we would like to include an example of this. Is it possible now?

Reply:
I don't know if it works. Its not obvious how to modify this example to do it, or what it would be appropriate in this context to include a handle to. A blob? The tree itself recursively? Make a different DDS and use that? A MockHandle? I dont have enough context to know whats appropriate.

Reply:
Another option would be to link a bug or issue here, so we could know if/when its fixed. I'm not working in that area, so I don't know off hand if the fix happened.

Reply:
@jikim-msft do we have a bug filed for this somewhere?

Reply:
As this PR doesn't impact this bug, I think we can improve or fix these todos in a separate change, though I'm going to try and find out the status of this.